tests/fixtures/rendering.yml

```diff
@@ -463,6 +463,33 @@
     equals:
       '.ecl-blockquote__author': "― Quote author"
       '.ecl-blockquote__body': "Quote text"
+- array:
+    '#type': pattern
+    '#id': context_nav
+    '#fields':
+      label: "Navigation title"
+      items:
+        - href: "#"
+          label: "Item one"
+        - href: "#"
+          label: "Item two"
+        - href: "#"
+          label: "Item three"
+        - href: "#"
+          label: "Item four"
+        - href: "#"
+          label: "Item five"
+      limit: 4
+      more_label: "More label"
+  assertions:
+    count:
+      'li.ecl-context-nav__item': 6
+      'li.ecl-context-nav__item--over-limit': 1
+      'li.ecl-context-nav__more': 1
+    equals:
+      '.ecl-context-nav__label': "Navigation title"
+      '.ecl-context-nav__button': "More label"
```

Comment:
Can you check the item labels and links as well?

Reply:
Done :)